src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ProfessionalInternalUserFunctionalTest.java

```diff
@@ -208,6 +209,18 @@ public void editPbaScenarios() {
     public void deleteOrganisationScenarios() {
         deletePendingOrganisationShouldReturnSuccess();
         deleteActiveOrganisationShouldReturnSuccess();
+        deleteDxAddressShouldReturnSuccess();
+    }
+
+    public void deleteDxAddressShouldReturnSuccess() {
+        log.info("deleteDxAddressShouldReturnSuccess :: STARTED");
+        String orgIdentifier = createAndUpdateOrganisationToActive(hmctsAdmin);
+        JsonPath jsonPath = professionalApiClient.retrieveOrganisationDetails(orgIdentifier, hmctsAdmin, OK);
+        String dxNumber = jsonPath.get("contactInformation[0].dxAddress[0].dxNumber");
+        String dxExchange = jsonPath.get("contactInformation[0].dxAddress[0].dxExchange");
+        professionalApiClient.deleteDXAddress(new DxAddressCreationRequest(dxNumber, dxExchange),
+                orgIdentifier, hmctsAdmin, NO_CONTENT);
+        log.info("deleteDxAddressShouldReturnSuccess :: END");
     }
```

Comment:
this is not a test .. need @Test above the method also run the test and make it pass add assert statements .check that the test covers all functionality by run with code coverage